inputs/manual_input.csv

```diff
@@ -318,12 +318,32 @@ offwind-float,FOM,2050,1.39,%/year,2020,https://doi.org/10.1016/j.adapen.2021.10
+lignite,fuel,2020,2.9,EUR/MWh_th,2010,"DIW (2013): Current and propsective costs of electricity generation until 2050, http://hdl.handle.net/10419/80348 , pg. 80 text below figure 10, accessed: 2023-12-14.","Based on IEA 2011 data, 10 USD/t."
+gas,fuel,2020,21.6,EUR/MWh_th,2010,"DIW (2013): Current and propsective costs of electricity generation until 2050, http://hdl.handle.net/10419/80348 , pg. 80 text below figure 10, accessed: 2023-12-14.",Based on IEA 2011 data.
 organic rankine cycle,lifetime,2020,30,years,2020,"Aghahosseini, Breyer 2020: From hot rock to useful energy: A global estimate of enhanced geothermal systems potential, https://www.sciencedirect.com/science/article/pii/S0306261920312551",
-organic rankine cycle,efficiency,2020,0.12,per unit,2020,"Aghahosseini, Breyer 2020: From hot rock to useful energy: A global estimate of enhanced geothermal systems potential, https://www.sciencedirect.com/science/article/pii/S0306261920312551; Breede et al. 2015: Overcoming challenges in the classification of deep geothermal potential, https://eprints.gla.ac.uk/169585/","This is a rough estimate, depends on input temperature, implies ~150 C"
+organic rankine cycle,electricity-input,2020,0.12,MWh_el/MWh_th,2020,"Aghahosseini, Breyer 2020: From hot rock to useful energy: A global estimate of enhanced geothermal systems potential, https://www.sciencedirect.com/science/article/pii/S0306261920312551; Breede et al. 2015: Overcoming challenges in the classification of deep geothermal potential, https://eprints.gla.ac.uk/169585/","Heat-input, Electricity-output. This is a rough estimate, depends on input temperature, implies ~150 C."
 organic rankine cycle,FOM,2020,2,%/year,2020,"Aghahosseini, Breyer 2020: From hot rock to useful energy: A global estimate of enhanced geothermal systems potential, https://www.sciencedirect.com/science/article/pii/S0306261920312551","Both for flash, binary and ORC plants. See Supplemental Material for details"
-organic rankine cycle,investment,2020,1500,USD/kW_el,2020,Tartiere and Astolfi 2017: A world overview of the organic Rankine cycle market,"Low rollout complicates the estimation, compounded by a dependence both on plant size and temperature."
-geothermal,CO2 intensity,2020,0.12,tCO2/MWh_el,2020,"Aghahosseini, Breyer 2020: From hot rock to useful energy: A global estimate of enhanced geothermal systems potential, https://www.sciencedirect.com/science/article/pii/S0306261920312551",Likely to be improved; Average of 85 percent of global egs power plant capacity
+organic rankine cycle,investment,2020,1376,EUR/kW_el,2023,Tartiere and Astolfi 2017: A world overview of the organic Rankine cycle market,"Low rollout complicates the estimation, compounded by a dependence both on plant size and temperature, converted from 1500 USD/kW using currency conversion 1.09 USD = 1 EUR."
```

Comment:
```suggestion
organic rankine cycle,investment,2020,1376,EUR/kW_el,2020,Tartiere and Astolfi 2017: A world overview of the organic Rankine cycle market,"Low rollout complicates the estimation, compounded by a dependence both on plant size and temperature, converted from 1500 USD/kW using currency conversion 1.09 USD = 1 EUR."
```

I think the currency year shouldn't be affected by the currency conversion. You're not adjusting for inflation with the conversion, right?

Reply:
Good point, adjusted!